### ISSUE TITLE
no-raw-colors reports fill-none as a raw color

### ISSUE THREAD

**Issue by gsmeira** [shadcn-ui/lint#54]
## Repro

With `shadcn/no-raw-colors` set to `error`, lint a component that uses the Tailwind `fill-none` utility on an SVG element:

```tsx
export function Ring() {
  return (
    <svg viewBox='0 0 36 36'>
      <circle className='fill-none stroke-primary' cx='18' cy='18' r='16' />
    </svg>
  )
}
```

The rule reports:

```
error shadcn(no-raw-colors): "fill-none" is not a declared theme color. Use one of: accent, background, border, card, chart-1, chart-2, chart-3, chart-4, chart-5, destructive, destructive-border, destructive-muted (+38 more). To add a color, declare --color-<name> in src/styles.css first.
```

## Why this is wrong

`fill-none` sets `fill: none`. `none` is the SVG paint keyword for "paint nothing", not a color, so no theme token can stand in for it. Suggesting `--color-<name>` sends users to declare a color they do not need.

The rule already accepts this keyword in its attribute check. `ATTRIBUTE_ALLOWED` in `dist/index.js` (line 5768 in 0.2.0) lists `currentColor`, `currentcolor`, `none`, `inherit`, `transparent`, `initial` and `unset`, and `isRawColorValue` returns `false` for these (line 5780). So `<circle fill="none" />` passes. The class check has its own keyword set, `NAMED` (line 5752: `white`, `black`, `transparent`, `current`, `inherit`), and `none` is not in it, so `className='fill-none'` fails. `fill-current` and `fill-transparent` already pass, so `fill-none` is the one keyword the two paths disagree on. `stroke-none` should be affected the same way (not tested).

## Suggested fix

Add `"none"` to `NAMED` in `packages/lint/src/rules/no-raw-colors.ts` (line 34 on `main`), so `fill-none` and `stroke-none` are treated like `fill-current` and `fill-transparent`.

## Workaround

Allow the class in the rule options: `'shadcn/no-raw-colors': ['error', { allow: ['fill-none'] }]`.

## Versions

- `@shadcn/lint` 0.2.0
- Oxlint 1.83.0 (JS plugin), via vite-plus 0.3.3
